examples/somersaultecu.py

```diff
@@ -1379,7 +1383,8 @@ class SomersaultSID(IntEnum):
                     description=None,
                     diag_coded_type=somersault_diagcodedtypes["uint8"],
                     byte_position=0,
-                    coded_value=SID.DiagnosticSessionControl.value,  # type: ignore
+                    coded_value=SID.DiagnosticSessionControl  # type: ignore[attr-defined,arg-type]
```

Comment:
instead of fighting mypy in those kind of cases, why not just define your own enum instead of using `SID`?

Reply:
the choice is between copying the SIDs or sprinkling the code with a few `type: ignore` comments- currently we follow the [latter approach](https://github.com/mercedes-benz/odxtools/blob/main/examples/somersaultecu.py#L83-L84).  the "correct" solution would be to somehow extend enum classes. (I'm not aware of a way of doing this can be done properly...)